adaptive_navigation/example/lib/default_scaffold.dart

```diff
@@ -20,7 +20,7 @@ class _DefaultScaffoldDemoState extends State<DefaultScaffoldDemo> {
     return AdaptiveNavigationScaffold(
       selectedIndex: 0,
       destinations: _allDestinations.sublist(0, _destinationCount),
-      appBar: AppBar(title: Text('Default Demo')),
+      appBar: AdaptiveAppBar(title: Text('Default Demo')),
```

Comment:
This might need to be a builder (or something similar) so the navigationType can be consumed and then set the leadingWidth to 72 when it needs to be 72 (only for nav rail?). Right now, it doesn't look like there is a way to switch this based on nav type.

Reply:
Done

Reply:
It does not look like this was addressed. Currently, the AdaptiveAppBar would have to know which NavigationType the AdaptiveNavigationScaffold is resolving to so it can set the correct leadingWidth